index.html

```diff
@@ -8,27 +8,27 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <div class="container" style="background-image:url('/home/ignatius/Documents/Microverse Projects/Portfolio/images/background-image.png');">
-        <div class="toolbar">
-            <p class="heading-1" style="color:blue ;">My logo</p>
+    <header class="container">
+        <nav class="toolbar">
+            <a href="#">My logo</a>
+            <img src="images/hamburger.png" alt="hamburger">
+        </nav>
+        <section class="greetings">
+            <p>I am Ignatius Cheruiyot. Glad to meet you!</p>
```

Comment:
As requested by the previous CR please replace it with the `h1 tag` for semantic best practices.